Tests/FoundationEssentialsTests/JSONEncoderTests.swift

```diff
@@ -1138,7 +1138,9 @@ final class JSONEncoderTests : XCTestCase {
         _test(JSONString: "[\"本日\"]", to: ["本日"])
     }
 
-    func test_JSONUnicodeEscapes() {
+    func test_JSONUnicodeEscapes() throws {
+        try XCTSkipIf(true, "current swift builds cause a stack overflow")
```

Comment:
Actually can you only skip for Linux? It believe it still works fine on other platforms.

Reply:
I didn't manage to test it on macOS - let's disable it only on Linux if it is passing on macOS.